src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/EventPublisherContainer.java

```diff
@@ -4,41 +4,47 @@
 import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 
-import javax.annotation.Resource;
-
 /**
  * Container class to hold availableStrategies strategies enabled by this project.
  * In order to enable one must do:
  * <ul>
  *     <li>implement {@link AbstractEventPublisher}</li>
- *     <li>include {@code @Resource(name = "publisher-name") private EventPublisher somePublisher;}</li>
+ *     <li>include {@code private EventPublisher somePublisher;}</li>
  *     <li>use resource in {@link this#getPublisher(Envelope, CaseDetails)}</li>
  * </ul>
  */
 @Component
 public class EventPublisherContainer {
 
-    @Resource(name = "attach-docs-to-supplementary-evidence")
-    private EventPublisher attachDocsPublisher;
+    private final EventPublisher attachDocsPublisher;
```

Comment:
I think we should rename it as it doesn't look like event publisher as either you update the case or create the case. What are your thought @doncem ?

Reply:
The naming was/is always a problem. Overall it is event (as per ccd resource endpoint) activity. Actual client method names are `startEventForCaseWorker` and `submitEventForCaseWorker` which we wrap with simple `startEvent` + `submitEvent`.. -> event publisher kind of thing. Can always rename to something clearer